config/routes.rb

```diff
@@ -6,6 +6,8 @@
   get '/sitemap.xml' => 'pages#sitemap', format: 'xml'
 
   get '/admin' => redirect('admin/dashboard')
+  get '/admin/sites/:id/select' => 'admin/change_site#index',
+        :as => :admin_select_site
```

Comment:
Style/AlignHash: Align the elements of a hash literal if they span more than one line.